app/proc/processor.go

```diff
@@ -105,6 +112,12 @@ func (p *Processor) feed(name, url, telegramChannel string, max int) {
 			continue
 		}
 
+		if (Filter{}) != filter {
+			skip, _ := filter.skip(item)
```

Comment:
ignoring error makes me nervous every time I see it. This is rarely justified and if it is - it needs a comment why. In this case, I see no reason to ignore